EnglishFilter/sections/specific.txt

```diff
@@ -260,6 +260,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 facebook.com##div[data-pagelet^="BrowseFeedUpsell_"] div[class][style^="max-width:"] > span
+earnflies.com##.box-main > p > a[href]
+earnflies.com##.box-main > p + a[href]
```

Comment:
maybe better without `.box-main`, because same buttons on the page after captcha
![image](https://user-images.githubusercontent.com/8361299/111307033-9b46f300-8661-11eb-8ebc-7d78c2ab52b3.png)

Reply:
As I said in another issue [href] doesn't narrow down. I'd go with `##p > a:only-child`

Reply:
Or
```
##.box-main p > a
##.box-main p + a
```
Actually `p + a` is unaffected.

Reply:
Don't need I think. The rules I have edited is enough and won't cause FPs.

Reply:
The current enough may not be enough for future. In any case unneeded `[href]` consumes memory.

Reply:
Okay I will change again.